zcutil/fetch-params.sh

```diff
@@ -4,19 +4,16 @@ set -eu
 
 PARAMS_DIR="$HOME/.zcash-params"
 
-REGTEST_PKEY_NAME='z9-proving.key'
-REGTEST_VKEY_NAME='z9-verifying.key'
-REGTEST_PKEY_URL="https://z.cash/downloads/$REGTEST_PKEY_NAME"
-REGTEST_VKEY_URL="https://z.cash/downloads/$REGTEST_VKEY_NAME"
-REGTEST_DIR="$PARAMS_DIR/regtest"
-
-# This should have the same params as regtest. We use symlinks for now.
-TESTNET3_DIR="$PARAMS_DIR/testnet3"
+BETA2_PKEY_NAME='beta2-proving.key'
```

Comment:
Non-blocking: These could just be named `PKEY_NAME` etc. to reduce size of future diffs.